### PR TITLE
Changes made after consult with Scott Russell to address the document…

### PR DESCRIPTION
**Pull request by timhowie-okta**
…ation for OKTA-446592

## Description:
- **What's changed?** Updated documentation to reflect changes in UI. Fixed typo (ASC -> ACS); fixed formatting of adding a variable (indented the list of information required per variable). 
- **Is this PR related to a Monolith release?** Yes, 2022.01.0

### Resolves:

* [OKTA-446592](https://oktainc.atlassian.net/browse/OKTA-446592)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md [hunk 1]
```diff
@@ -28,10 +29,12 @@
...
 
+* **Okta requires that all OIDC integrations support multi-tenancy. Do you support this in your implementation?** &mdash; select **Yes** if your customers can set up an OIDC connection between their tenant in your application and multiple external IdPs or even multiple instances inside a single IdP.
+
 * **To configure OIDC, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; if a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the setup time for your customers.
 
 * **Link to configuration guide**: Your configuration guide (in either HTML or PDF format) should have step-by-step instructions on how to configure SSO between Okta and your systems. See [Prepare a customer-facing configuration guide](/docs/guides/submit-app/openidconnect/main/#prepare-a-customer-facing-configuration-guide).
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:36]: Why have we switched from an mdash to a colon, which continues on the following two questions. Should stick with mdash for consistency thoroughout

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md [hunk 2]
```diff
@@ -15,6 +15,7 @@
   If you're using a per tenant design, include the variable names that you created. For example:
   * https://`${app.variableName}`.okta.com
   * https://okta-`${app.variableName}`.com
+  * `${app.variableName}`
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:18]: Is this valid to not have an `https://` at the start?

**Review by IanHakes-okta — APPROVED** [review]
One technical question to confirm (which appears in 3 places), and a punctuation consistency to clear up. 

Approving so I won't halt the merge when those issues have been addressed.

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md (hunk 1 above)
Comment (timhowie-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:36]: Good catch - thanks. At the time it made sense, for what it's worth. :)

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md (hunk 2 above)
Comment (timhowie-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:18]: Yes - this is the point of this change; the variable can now include http:// | https:// as part of the variable, which was requested by customers. I'll check with him if we should also use `${app.variableName}`/route here to better indicate this. The release note will point out that the variables for dynamic redirect URIs can now include this

Comment (ScottRussell-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:20]: @IanHakes-okta for context: OPS asked us to allow a full URL as variable, which makes something like Redirect URL or ACS URL etc validation much simpler. https/.com are no longer required -- so long as a variable exists -- in order to allow customers to utilize more globally useful variables such as ${org.baseUrl}; I think we will commonly see these inside of a dynamic url input like: ${org.baseURL}/login etc.  Unfortunately, this doesn't work very well along side the previous examples.

**Review by ScottRussell-okta — APPROVED** [review]
LGTM.

I'll add that `${app.variableName}/route` vs `${app.variableName}` in the examples additions is fine with me either way. The regex change is meant to allow dynamic url inputs to contain both variable as _part_ of a URL (subdomain only, etc) as well as a qualified URL itself, so I think that either is fine. My intuition was that it would commonly present with the fully qualified domain as variable but routes like /signout or /login/callback etc may be added after the variable in dynamic url inputs.

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md (hunk 2 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md]: Maybe we need to add some text here to explain that?

> If you're using a per tenant design, include the variable names that you created. You can use a full URL as a variable. This enables you to use more globally useful variables such as `org.baseURL`. The `https://` and `.com` values are not strictly required.
 
 For example:
(etc.)

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md [hunk 3]
```diff
@@ -1,37 +1,42 @@
...
+* **Construct your dynamic Redirect URI by copying the variables above and pasting them where applicable**: Provide one or more complete sign-in redirect URIs where Okta sends the OAuth responses for your app integration. You must add at least one valid redirect URI.
 
   If you're using a per tenant design, include the variable names that you created. For example:
   * https://`${app.variableName}`.okta.com
   * https://okta-`${app.variableName}`.com
+  * `${app.variableName}/route`
 
-* **Optional: Do you require an Initiate Login URI** &mdash; include a login URI if you want the Okta integration to handle redirecting your users to your application to start the sign-in request. When end users click a tile in their Okta dashboard, they are redirected to the `initiate_login_uri` of the client application, which constructs an authorization request and redirects the end user back to Okta.
+  > **Note**: A variable can include a complete URL (for example, https://example.com). This enables you to use more globally useful variables such as `${org.baseURL}`.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:20]: @ScottRussell-okta : I moved the note below the variable examples and added $,{,} around org.baseURL. Please confirm if this edit is OK. Thanks

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md [hunk 4]
```diff
@@ -1,73 +1,76 @@
...
 
-* **Do you support SP-initiated Single-Logout?** &mdash; if you select **Yes**, then **Are you going to use the same single-logout certificate for all tenants?**
+* **Do you support SP-initiated Single-Logout?**: If you select **Yes**, then **Are you going to use the same single-logout certificate for all tenants?**
 
     If you support the same single-logout certificate for all tenants, copy your certificate and paste it in the field provided.
 
     The Single Logout URL and SP Issuer should be specified in the test application.
 
-    >**Note:** Okta only supports signed logout requests.
+    >**Note:** Okta only supports single logout requests.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md:68]: @ScottRussell-okta : I updated "signed" to "single" in this note - please confirm if this correction. Thanks

**Review by vanngo-okta — COMMENTED** [review]
A couple of questions for @ScottRussell-okta to confirm. Thank you!

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md (hunk 3 above)
Comment (ScottRussell-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:20]: This looks good to me.  Just to verify, the note section is added to Oidc/Saml/Scim tab's `submit.md` equally?

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md (hunk 4 above)
Comment (ScottRussell-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md:68]: Good catch, thank you.  Yes that is correct.

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:20]: Yes, the same note is in all 3 `submit.md` files.

**Review by susanharper-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md [hunk 5]
```diff
@@ -1,37 +1,42 @@
 #### OIDC settings
 
-* **Does your Redirect URI vary per tenant?** &mdash; if **Yes**, a new field appears to assist you in setting up a per tenant configuration.
-  * **What variables do your admins need to specify to install your app?** &mdash; when you click **Add Variable**, the interface displays a dialog box to collect the following information:
-  * **Label Name** &mdash; a descriptive name for the dynamic variable that administrators see when installing your app integration.
-  * **Variable Name** &mdash; an automatically generated variable used when constructing the dynamic address. This is hidden from admins and is only passed to your external application.
-  * **Help Text** &mdash; any descriptive text to be shown to administrators when installing your app integration.
-  * **Type** &mdash; the property type for your parameter. Options are "String", "URL", or "HTTPS URL".
+* **Does your Redirect URI vary per tenant?**: If **Yes**, a new field appears to assist you in setting up a per tenant configuration.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:3]:
```suggestion
* **Does your Redirect URI vary per tenant?**: If **Yes**, a new box appears to assist you in setting up a per tenant configuration.
```

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md [hunk 6]
```diff
@@ -1,37 +1,42 @@
...
 
-* **Enter your Post Logout Redirect URI** &mdash; provide the complete sign-out redirect URI for your app integration, including the variable names if you're using a per tenant design.
+* **Does your Post Logout URI vary per tenant?**: If **Yes**, an additional field appears to assist you in setting up a per tenant configuration. The instructions for adding and using the custom variables are identical to the per tenant instructions for the Redirect URI.
 
-* **To configure OIDC, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; if a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the setup time for your customers.
+* **Enter your Post Logout Redirect URI**: Provide the complete sign-out redirect URI for your app integration, including the variable names if you're using a per tenant design.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md:32]:
```suggestion
* **Enter your Post Logout Redirect URI**: Provide the complete sign-out direct URI for your app integration, including the variable names if you're using a per tenant design.
```

**Pull request merged**